decidim-core/lib/decidim/data_portability.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require "active_support/concern"
+
+module Decidim
+  # This concern contains the logic related to data portability.
+  module DataPortability
+    extend ActiveSupport::Concern
+
+    included do
+      def self.user_collection(user)
```

Comment:
Please add documentation about the expectations of this method, as it needs to be overriden by its implementors.